app/models/network/node.rb

```diff
@@ -1,5 +1,7 @@
 module Network
   class Node < Turbine::Node
+    include BusinessCaseCosts
```

Comment:
If what you want to do it take a list of nodes and ask them for their costs, while that is not the responsibility of a *Network::Node*; the [Decorator pattern](https://en.wikipedia.org/wiki/Decorator_pattern) would allow you to do this while keeping your other code simple:

```ruby
class NodeCostsDecorator
  include BusinessCaseCosts

  def initialize(node)
    @node
  end

  def initial_investment
    # ...
  end

  def om_costs_per_year
    # ...
  end

  # ...
end
```

Then, instead of using the Network::Node instances directly, you wrap each node in an Decorator and use those instead:

```ruby
nodes = network.nodes.map { |node| NodeCostsDecorator.new(node) }
```

You can see this in action in TestingGround::HeatSummary [where TechnologySummary](https://github.com/quintel/etmoses/blob/7a9c55e4b1b0da1c5fb8e0bd084e67e0824c6631/app/services/testing_ground/heat_summary.rb#L172-L197) takes `Network` objects, and then create a new interface for use in the HeatSummary without polluting the original class with new methods.